### DISCUSSION TITLE
Can I  run this project with docker or k8s ?

### DISCUSSION THREAD

**Discussion by samzong** [illacloud/illa-builder#338] [Q&A] [answered]
Can not find getting-started.

**Comment by naj1n** [comment]
We recommend using ILLA CLI to run a self-hosted installation of ILLA Builder. For more information, please visit [ILLA CLI doc](https://www.illacloud.com/docs/illa-cli).

**Reply by anson0370** [reply]
The ILLA CLI is easy and fast to run a self-hosted ILLA Builder. 👍 
But I can not login, did not find default username/password in doc. Register new account failed because it got a 500 error when I tried to get the verify code.

**Reply by naj1n** [reply]
@anson0370 please provide a little bit more details on this error you got. What API request did ILLA Builder run? And what this request outputs did you receive? Thanks

**Reply by anson0370** [reply]
<img width="1512" alt="image" src="https://user-images.githubusercontent.com/283559/188816764-40405b43-1f07-428c-a844-36b2f1c82ffb.png">
FYI. There is no response for this request. Where is the backend log file in the container? Maybe I can find the error stack.

What I did is fill the email and click the get verify code link.
<img width="789" alt="image" src="https://user-images.githubusercontent.com/283559/188817324-c7158dc7-23bf-477c-9bbb-13419833d3e5.png">

**Reply by naj1n** [reply]
Emm, yeah. I want to know where the ILLA Builder is deployed. PC or Cloud server?

**Reply by anson0370** [reply]
On my MBP, it's a M1 chip MBP. Maybe the CPU arch is the problem? I noticed the illasoft/illa-builder image has a amd64 arch.

**Reply by naj1n** [reply]
Nope, the CPU arch isn`t the problem. I will follow these given steps to reproduce the output. Wait a minute.

**Reply by naj1n** [reply]
sry, I can`t reproduce the output. Could u join our [discord channel](https://discord.com/invite/2tGBuJkgd6) for better communication?

**Reply by naj1n** [reply]
Another way that may be helpful to this problem is to run `illa remove -f --self -i=<ILLA_BUILDER_ID> `, it will remove the deployed ILLA Builder. And then, u can run `illa deploy --self --port=10000` to deploy a new one.

**Reply by anson0370** [reply]
Will join later, thanks for reply.

**Comment by karminski** [comment, marked as answer]
@anson0370 

Now we support the docker and k8s deploy!

Please check this: [https://github.com/illacloud/deploy-illa-manually](https://github.com/illacloud/deploy-illa-manually)
